plugins/kubernetes-backend/src/service/KubernetesBuilder.ts

```diff
@@ -262,12 +271,20 @@ export class KubernetesBuilder {
     clusterSupplier: KubernetesClustersSupplier,
     catalogApi: CatalogApi,
     proxy: KubernetesProxy,
+    permissionApi: PermissionEvaluator,
   ): express.Router {
     const logger = this.env.logger;
     const router = Router();
-    router.use('/proxy', proxy.createRequestHandler());
+    router.use('/proxy', proxy.createRequestHandler({ permissionApi }));
     router.use(express.json());
-
+    router.use(
+      createPermissionIntegrationRouter({
+        resourceType: RESOURCE_TYPE_KUBERNETES_PROXY_RESOURCES,
+        permissions: kubernetesPermissions,
+        rules: [],
+        getResources: _ => Promise.resolve([]),
```

Comment:
with #16333 merged you don't need this line

Reply:
Awesome Thanks. Removed the getResources parameter from the createPermissionIntegrationRouter. Don't think there were any other suggestions for this route right?